.travis.yml

```diff
@@ -31,19 +29,16 @@ before_install:
 - hash -r
 #- conda update --yes conda
 - conda info -a
-- conda install --yes python=$PY numpy scipy cython pylint wxpython matplotlib lxml h5py sphinx pyparsing html5lib reportlab pybind11 appdirs six
+- conda install --yes python=$PY numpy scipy cython pylint wxpython matplotlib lxml h5py sphinx pyparsing html5lib reportlab pybind11 appdirs six ipython qtconsole
 install:
-- pip install periodictable xhtml2pdf pyopencl unittest-xml-reporting pylint
+- pip install periodictable xhtml2pdf pyopencl unittest-xml-reporting pylint qt5reactor
 script:
 - cd ..
 - export WORKSPACE=$(pwd)
-- git clone --depth=50 --branch=master https://github.com/SasView/sasmodels.git sasmodels
+- git clone --depth=50 --branch=tmp/doc-build https://github.com/llimeht/sasmodels.git sasmodels
```

Comment:
Should reference SasView/sasmodels github.

Reply:
Of course. That's one of the reasons that this is a PR draft that GitHub won't let you merge, rather than a PR. That just working around a resolution to https://github.com/SasView/sasmodels/pull/373 .